src/main/java/org/fungover/thunder/Main.java

```diff
@@ -1,16 +1,15 @@
 package org.fungover.thunder;
 
 import java.io.IOException;
+import java.net.ServerSocket;
 
 public class Main {
 
-    public static void main(String[] args) {
-        Server server;
-        try {
-            server = new Server();
-            server.start();
-        } catch (IOException | NullPointerException e) {
-            e.printStackTrace();
-        }
+    public static void main(String[] args) throws IOException {
+        ServerSocket serverSocket = new ServerSocket(1883);
+        ClientHandler clientHandler = new ClientHandler();
+        Broker broker = new Broker(clientHandler, serverSocket);
+        broker.start()
```

Comment:
Missing a semicolon ';' at the end of broker.start()

Reply:
Never mind 😂

Reply:
Thanks, I have now :-)